android_symbol_uploader.go

```diff
@@ -142,14 +141,11 @@ func (a *androidSymbolUploader) getBuildVersion(e *env) (int, error) {
 		return versionCode, nil
 	}
 
-	versionCode, err := a.getBuildVersionFromAPK()
-	if err != nil {
-		return 0, err
-	}
-	if versionCode == 1 {
-		fmt.Fprintln(e.Out, "Warning: build number is '1'.")
+	if a.Manifest != "" {
+		return handleError(a.getBuildVersionFromManifest())
+	} else {
```

Comment:
you do not need an else.
if we enter if then we just return

so 
if {
return
}
return

is good. this is idiomatic go :)